typescript-json-schema.ts

```diff
@@ -854,6 +866,16 @@ export class JsonSchemaGenerator {
         return root;
     }
 
+    public getSymbols(): SymbolRef[];
```

Comment:
What does this method do here?

Reply:
Thought it may be more clear to use function overloading here. Could also be removed, since the implementation with the optional parameter should do.